src/util/String.h

```diff
@@ -97,6 +97,16 @@ class String {
         }
         return ss.str();
     }
+
+    /**
+     * Encode the input sequence of bytes into the hexadecimal representation packaged
+     * into a string.
+     * @param ptr A pointer to the byte sequence.
+     * @param length The number of bytes to translate.
+     * @return The encoded sequence of bytes or the empty string if the length=0.
+     * @throw std::invalid_argument If the pointer is nullptr.
+     */
```

Comment:
I think an example of input to output would help make it clearer what this is doing. From the jira, it's converting `[10,17,255,210]` to `"0A11FFD2"`

Reply:
Good suggestion!